sigstore-java/src/main/java/dev/sigstore/trustroot/LegacySigningConfig.java

```diff
@@ -48,16 +49,25 @@ static SigstoreSigningConfig from(URI fulcioUrl, URI rekorUrl, URI dexUrl, @Null
             .tLogConfig(anySelector)
             .tsaConfig(anySelector)
             .addCas(Service.of(fulcioUrl, 1))
-            .addTLogs(Service.of(rekorUrl, 1))
             .addOidcProviders(Service.of(dexUrl, 1));
+
+    if (rekorUrl.equals(REKOR_V2_STAGING_URI)) {
+      signingConfigBuilder.addTLogs(Service.of(rekorUrl, 2));
+    } else {
+      signingConfigBuilder.addTLogs(Service.of(rekorUrl, 1));
+    }
+
```

Comment:
I think we should do this another way. `from(...)` is meant to be generic without knowlege of URIs. We can change the inputs to be Service instead of URI?